cluster/src/main/java/org/apache/iotdb/cluster/utils/RaftUtils.java

```diff
@@ -302,4 +314,87 @@ public static Status createErrorStatus(String errorMsg){
     status.setCode(-1);
     return status;
   }
+
+  public static ConcurrentHashMap<String, PeerId> getGroupLeaderCache() {
+    return groupLeaderCache;
+  }
+
+  public static Map<Integer, String> getPhysicalRing() {
+    SortedMap<Integer, PhysicalNode> hashNodeMap = router.getPhysicalRing();
+    Map<Integer, String> res = new LinkedHashMap<>();
+    hashNodeMap.entrySet().forEach(entry -> res.put(entry.getKey(), entry.getValue().getIp()));
```

Comment:
ip+port is better?

Reply:
I consider this before, but all other tools only output ip, like Cassandra. As the port is rarely changed, it should be unnecessary to output port.